tutorials/cloudsdk-integrate-cap/cloudsdk-integrate-cap.md

```diff
@@ -16,25 +16,21 @@ author_profile: https://github.com/MatKuhr
  - An account on [Cloud Foundry](group.scp-1-get-ready)
 
 
-
-## Intro
 > ## We migrate tutorials to our [documentation](https://sap.github.io/cloud-sdk/)
```

Comment:
Do not header formatting to emphasize text. \*\*We migrate...\*\* would be correct in this case.